files/bootstrap.sh

```diff
@@ -430,8 +430,11 @@ set +o pipefail
 MAX_PODS=$(cat $MAX_PODS_FILE | awk "/^${INSTANCE_TYPE:-unset}/"' { print $2 }')
 set -o pipefail
 if [ -z "$MAX_PODS" ] || [ -z "$INSTANCE_TYPE" ]; then
-    echo "No entry for type '$INSTANCE_TYPE' in $MAX_PODS_FILE"
-    exit 1
+    echo "No entry for type '$INSTANCE_TYPE' in $MAX_PODS_FILE. Will attempt to auto-discover value."
+    # When determining the value of maxPods, we're using the legacy calculation by default since it's more restrictive than
+    # the PrefixDelegation based alternative and is likely to be in-use by more customers.
+    # The legacy numbers also maintain backwards compatibility when used to calculate `kubeReserved.memory`
+    MAX_PODS=$(./max-pods-calculator.sh --instance-type-from-imds --cni-version 1.10.0 --show-max-allowed)
```

Comment:
I believe this needs to be absolute path or relative to the script dir.

Reply:
It's in the same directory as the bootstrap script which is why I left it this way. I can make this an absolute path.

Reply:
The path is relative to the location of the invoker, not the script itself. If you wanted to construct an absolute path to the dir where the script is located, you'd need something like this:

```
SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
```

Reply:
Whoops yes my bad